src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -206,10 +206,11 @@ class TaskLauncherActorTest extends AkkaUnitTest {
         .setOperator(Operator.UNIQUE)
         .setValue("")
         .build
-      val constraintApp: AppDefinition = f.app.copy(constraints = Set(uniqueConstraint))
+      val unreachableStrategy = UnreachableEnabled(5.minutes, 10.minutes)
```

Comment:
Do we have a test for (0,0) defaults?

Reply:
Kind of. We do test this in an integration test: https://github.com/mesosphere/marathon/blob/0f471239370d702ce2b042a4c8cb7b8ff837f952/src/test/scala/mesosphere/marathon/integration/TaskUnreachableIntegrationTest.scala#L90

To be sure, I`m going to a unit test to cover this!

Reply:
Awesome! Thanks @unterstein!